LCM/Code/App/task.c

```diff
@@ -553,33 +552,22 @@ void Power_Task(void)
 void CheckPowerLevel(float battery_voltage)
 {
 	#ifdef P42A
-	float battVoltages[10] = {4.054, 4.01, 3.908, 3.827, 3.74, 3.651, 3.571, 3.485, 3.38, 3.0}; //P42A
+	float battVoltages[11] = {4.054, 4.01, 3.908, 3.827, 3.74, 3.651, 3.571, 3.485, 3.38, 3.0, 2.7}; //P42A
```

Comment:
this doesn't look right. i think 4.2 should be prepended to these arrays, instead of appending a low value.

Reply:
It's correct in that it treats the first entry as 100% full. Most BMSs balance down to a voltage below 4.2V, so these values allow for a display of 100% at the balance voltage. 

That said, with the dimming capability that this PR provides, there's really no reason to not use 4.2V as the upper bound since that high order LED will be mostly illuminated when it's at the slightly lower balance voltage.

Reply:
No, the old code treated the range 4.2 to battVoltages[0] as 10 leds (as evidenced by the comment at https://github.com/surfdado/floatwheel/pull/23/files#diff-b7f0798cc0902af54faf8df77a07a16df8bde4b47e7471a75cdec8be5f1d7562L34 Power_Display_Flag = 1�� 4.08V~4.2V 10���׵�), the new code treats the range battVoltages[0] to battVoltages[1] as 100% to 90%

Reply:
I agree that we need to prepend 4.2 to the arrays. However, they should probably also be adjusted such that 3.0 is the lowest value per your other comment. I'll take a closer look at this and compare these values to those used by Floaty for it's percentage tables.